src/Native/Windows/CMakeLists.txt

```diff
@@ -91,16 +91,9 @@ endif()
 
 # App-Container builds require a more specific linking to ensure the APIs imported pass WACK
 if ($ENV{__appContainer} STREQUAL true AND UPPERCASE_CMAKE_BUILD_TYPE STREQUAL "RELEASE")
-    set(__LinkCustomCRT true)
     set(__LinkArgs "${__LinkArgs} /LTCG")
```

Comment:
`/LTCG` option is added a few lines below for all build flavors: https://github.com/dotnet/corefx/pull/30256/files#diff-5822bf65a2871321bce04bde060ce1a0L122

We should not need to duplicate it here.

Reply:
Removed